pkg/secrets/provide_conjur_secrets_test.go

```diff
@@ -143,7 +144,7 @@ func TestSecretProvider(t *testing.T) {
 			description:   "sidecar container, happy path",
 			mode:          "sidecar",
 			interval:      time.Duration(10) * time.Millisecond,
-			testTime:      time.Duration(65) * time.Millisecond,
+			testTime:      time.Duration(65+atomicWriteDelayMsecs) * time.Millisecond,
```

Comment:
What's all this about ? (Just curious)

Reply:
In the case where we're doing atomic writes, it adds a very small delay which causes the retry testing to get thrown off (though only on Jenkins, not when running locally). This fixes that.